js/level1.js

```diff
@@ -4,34 +4,35 @@
     Comments
     ========
 
-    Let's start with comments. Comments are notes that people can read and
-    computers will ignore.
+    Let's start with comments. This is a comment. Comments are notes that people
+    can read and computers will ignore.
 
     They will help us to guide you through the JavaScript introduction
     journey.
 */
 
+// Single line comments look like this.
+
 /*
     Multi-line comments look like this.
+    When you are writing actual code, put it outside these comment blocks,
+    so the computer doesn't ignore them.
 */
 
-// Single line comments look like this.
-
-
 /*
     Let's start with getting your code on the screen.
 
     There are a few ways you can do it and we will look into a few of them:
 
-    * alert('Hello girls!');
+    alert('Hello girls!');
```

Comment:
People usually write things with the *, like `*alert('Hello girls!');`.

`*` is not used anywhere else in these files; so hopefully this makes it clearer.